types/steamcommunity/index.d.ts

```diff
@@ -376,22 +376,22 @@ declare namespace SteamCommunity {
          * An instance of {@link https://www.npmjs.com/package/request|request} v2.x.x which will be used by `SteamCommunity` for its HTTP requests.
          * SteamCommunity` will create its own if omitted.
          */
-        request: Request;
+        request?: Request;
```

Comment:
It might be worth explicitly adding `| undefined` to these optionals for the `exactOptionalPropertyTypes` config option: <https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes>

```suggestion
        request?: Request | undefined;
```